client/components/account-status/account-fees.js

```diff
@@ -0,0 +1,80 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement as createInterpolateElement } from 'wordpress-element';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrency, formatCurrency } from 'utils/currency';
+
+const AccountFees = ( props ) => {
```

Comment:
Same as with account status component, it's very easy to mix up with `client/account-fees/index.js`.

Reply:
It's a temporary nuiance. AFAIK it is expected that `client/account-fees` will cease to exist once account overview page is fully functional.

Reply:
I changed them to `account-fees-settings` and `account-status-settings` to make them more distinctive.

Reply:
Nice!!!